Formula/b/bounceback.rb

```diff
@@ -0,0 +1,36 @@
+class Bounceback < Formula
+  desc "Stealth redirector for red team operation security"
+  homepage "https://github.com/D00Movenok/BounceBack"
+  url "https://github.com/D00Movenok/BounceBack/archive/refs/tags/v1.4.2.tar.gz"
+  sha256 "9f18c3b00f19f832a6ec9cfc4383d38a3ca400e470cc356c3cd41ea3d7919466"
+  license "MIT"
+  head "https://github.com/D00Movenok/BounceBack.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X main.version=#{version}
+    ]
+    system "go", "build", *std_go_args(ldflags: ldflags), "./cmd/bounceback"
+
+    pkgshare.install "config.yml"
+    pkgshare.install "data"
+  end
+
```

Comment:
it might be better that we can also add `service` block in here. (looks like a server app)